book/website/ethical-research/activism/activism-env-impact.md

```diff
@@ -0,0 +1,168 @@
+* If you do test-driven development, run only tests that have previously failed. In this way you do not need to test code which you already know has been tested. You can use this GH action for limiting when tests (pytest) are run: [https://github.com/marketplace/actions/pytest-last-failed](https://github.com/marketplace/actions/pytest-last-failed) 
+* Run CI with smaller datasets.
+
+(er-activism-env-impact-schedule-low-emission)=
+#### Schedule tasks at low-emission time
+
+Energy usage at different times of the day has different carbon intensity. This means that there is also an opportunity to reduce carbon emissions by running computing jobs at different times of the day. While the energy usage remains the same, the carbon intensity can be lowered in this way.
+
+The Climate Aware Task Scheduler (CATS)[https://github.com/GreenScheduler/cats] has been built specifically with this in mind. This tool can calculate how much carbon will be emitted during the run of a specific task, look at the carbon emission forecast, and schedule the task to be run at a time when carbon intensity is low.
+
+```{figure} ../../figures/carbon_computing.jpg
+---
+height: 500px
+name: Computing has carbon emissions illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
+alt: Computing has carbon emissions
+---
+Computing has carbon emissions
+```
+
+(er-activism-env-impact-data)=
+### Data
+The increasing number of digital research and the associated storage requirements have implications for the environment, and understanding the environmental impact is key for sustainable scientific research practices. One important aspect to consider is the environmental impact of digital storage:
+
+(er-activism-env-impact-data-reduction)=
+#### Data reduction 
+Reduction of data volumes is a straight-forward solution to minimize energy consumption in storage systems:
+* Compression or deduplication can help to detect and delete repeated information. 
+* Optimized data requires less time for transfer and consumes less network bandwidth.
+
+(er-activism-env-impact-data-standardisation)=
+#### Standardisation
+Storing your data in standardised data format can have a positive impact on the environment:
+* Utilize data formats that are widely accepted within your community to prevent the need for conversion by those interested in reusing your data.
+* Similarly, embrace standardized variable names and, if applicable to your data, employ standardized physical units.
+
+In addition, deposit your datasets in domain specific or community archives so that users ware promptly informed about the availability of your dataset. This can also avoid duplication of effort and recomputation/acquisition of the same data.

```

Comment:
```suggestion
In addition, if possible deposit your datasets in domain specific or community archives so that users are promptly informed about the availability of your data. This can also avoid duplication of effort and recomputation/acquisition of the same data.

```